guide/14-deep-learning/pixel_based_classification.ipynb

```diff
@@ -79,14 +79,20 @@
    "cell_type": "markdown",
```

Comment:
<p>Notes:</p><ul><li>The raster and labeled imagery layer both are required to be '8 bit Unsigned' when the raster is a RGB layer such as world imagery.</li><li>The Labeled imagery layer should be a thematic raster with pixel values corresponding to the label class value.</li><li>The pixel values should range <strong>from from</strong> 1 to n, where n is the total number of classes.</li><li>Any <strong>nodata</strong> value should be mapped to 0, as portions of image with nodata values would be ignored in while exporting <strong>trianing</strong> data.</li></ul><p>Comment: </p><ul><li>The second bullet has redundant word 'from', last bullet has incorrect spelling which is in bold. </li><li>I am not sure if nodata should be no data, is it a specific keyword used?</li></ul>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/574/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>

Reply:
<p>i have changes nodata to NoData</p><p><a href="https://desktop.arcgis.com/en/arcmap/10.3/manage-data/raster-and-images/nodata-in-raster-datasets.htm" target="_blank">https://desktop.arcgis.com/en/arcmap/10.3/manage-data/raster-and-images/nodata-in-raster-datasets.htm</a></p>